runtime/bindings/python/iree/runtime/system_setup.py

```diff
@@ -25,9 +25,16 @@ def query_available_drivers() -> Collection[str]:
     return HalDriver.query()
 
 
-def get_driver(device_uri: str) -> HalDriver:
-    """Returns a HAL driver by device_uri (or driver name)."""
-    return get_cached_hal_driver(device_uri)
+def get_driver(device_uri: str, clean: bool = False) -> HalDriver:
```

Comment:
Let's switch from `clean = False` to `use_cache = True`. I'd still like the API to actually reset the cache to be private (as in accessible but slightly hard to get to).

Reply:
This discussion is made obsolete by the change to the 3-function API.